proto/ibc/core/connection/v1/tx.proto

```diff
@@ -88,22 +78,15 @@ message MsgConnectionOpenAck {
 
   option (gogoproto.goproto_getters) = false;
 
+  reserved 4, 7 to 9, 11;
```

Comment:
this will break e2e's unfortunately since the sdk (or proto, not entirely sure which) will reject msg submissions which have values set for reserved fields. We would need an updated relayer binary to stop filling in this fields. I was thinking maybe we just leave as is for now and reserve the fields once relayers are updated? (ie deprecate)

Reply:
Yep. Reverted it.